lib/vendors/axis-communications/sv_vendor_axis_communications_internal.h

```diff
@@ -150,4 +150,13 @@ get_axis_communications_supplemental_authenticity(void *handle,
 const char *
 get_axis_communications_trusted_certificate(void);
 
+/**
+ * @brief Retrieves the certificate chain from the vendor handle.
+ *
+ * @param sv The signed_video_t structure containing vendor_handle.
+ * @return The certificate chain as a null-terminated string, or NULL on failure.
+ */
+const char *
+get_certificate_chain(signed_video_t *sv);
```

Comment:
The name of this function should be `get_axis_communications_certificate_chain` to follow the name convention of the other function names.